builtin/providers/spotinst/resource_spotinst_aws_group.go

```diff
@@ -0,0 +1,1837 @@
+package spotinst
+
+import (
+	"bytes"
+	"crypto/sha1"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+)
+
+func resourceSpotinstAwsGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSpotinstAwsGroupCreate,
+		Read:   resourceSpotinstAwsGroupRead,
+		Update: resourceSpotinstAwsGroupUpdate,
+		Delete: resourceSpotinstAwsGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
```

Comment:
Just a point here, we don't need to set `ForceNew: false` - this is the default :)